data_management/mysqldumpall/dumpall.sh

```diff
@@ -73,29 +81,29 @@ for database_name in $database_names; do
 
   # Get all table names from database file
   echo Get all table names from database file
-  tables=$(grep --text -P '^-- Table structure for table ' "$target_dir"/"$database_name"-temp/"$database_name".sql | awk '{ print $NF }' | sed 's/`//g');
+  tables=$(grep --text -P '^-- Table structure for table ' "$tmp_dir"/"$database_name"-temp/"$database_name".sql | awk '{ print $NF }' | sed 's/`//g');
 
   for table in $tables; do
     # Extract table
     echo Extract table $database_name.$table
-    $mysqldumpsplitter_command --source "$target_dir"/"$database_name"-temp/"$database_name".sql --extract TABLE --match_str "$table" --compression none --output_dir "$target_dir"/"$database_name"-temp/tables-temp >&2
+    $mysqldumpsplitter_command --source "$tmp_dir"/"$database_name"-temp/"$database_name".sql --extract TABLE --match_str "$table" --compression none --output_dir "$tmp_dir"/"$database_name"-temp/tables-temp >&2
 
     # Remove comments starts with -- in table sql
     echo Remove comments starts with -- in table sql $database_name.$table
-    grep -vw --text '^--' "$target_dir"/"$database_name"-temp/tables-temp/"$table".sql > "$target_dir"/"$database_name"/tables/"$table".sql
+    grep -vw --text '^--' "$tmp_dir"/"$database_name"-temp/tables-temp/"$table".sql > "$target_dir"/"$database_name"/tables/"$table".sql
   done
 
   # Remove comments starts with -- in the main database sql 
   echo Remove comments starts with -- in the main database sql $database_name
-  grep -vw --text '^--' "$target_dir"/"$database_name"-temp/"$database_name".sql > "$target_dir"/"$database_name"-temp/database.sql
+  grep -vw --text '^--' "$tmp_dir"/"$database_name"-temp/"$database_name".sql > "$tmp_dir"/"$database_name"-temp/database.sql
```

Comment:
Is  this line actually necessary, considering what happens with `grep` on the next line?

Reply:
I removed this line since this is not necessary.